### ISSUE TITLE
Files stops responding when a specific ftp folder is browsed. 

### ISSUE THREAD

**Issue by yurutken** [elementary/files#1331]
When i connect a ftp and browse to a specific folder with jpg images Files stops responding.
Video is attached. It shows a file name at right bottom and locks immediately and stops responding.
Another browser for ex (nautilus ) can show folder contents without problem. 
Folder contents and lshw is attached.

Possible Cause : Folder name starts with a Turkish character. But other browsers seem ok with that.
[lsal.txt](https://github.com/elementary/files/files/4651347/lsal.txt)
[lshw.txt](https://github.com/elementary/files/files/4651348/lshw.txt)

[video.zip](https://github.com/elementary/files/files/4651368/video.zip)

[HardInfoSystemReport.pdf](https://github.com/elementary/files/files/4651404/HardInfoSystemReport.pdf)

**Comment by jeremypw** [comment]
I did not have a problem using a "FTP with login" connection and with thumbnailing enabled and with one test jpg.  What type of FTP connection do you have? Are you using remote thumbnailing?  Does it still hang with remote tumbnailing turned off?    Does it hang immediately or only after hovering an image with the pointer?

**Comment by yurutken** [comment]
Hello!
1. It doesnt hang at all if i dont hover an image with pointer. It hangs just after any of image is being hovered.
2. Remote thumbnailing was off. I turned it on - off even local thumbnailing on and off. It still hangs. I think it is not related to thumbnailing. If i dont hover pointer actually remote thumbnailing works just fine. Problem caused by hovering mouse to a file.
3. I can send you related ftp username and password privately ?

**Comment by jeremypw** [comment]
It sounds like a problem with the Overlaybar which shows image info - it appears to be blocking indefintely trying to determine the image dimensions.  On my test setup there was no problem with analysing the test jpg over FTP.  If possible could you download one of the problem images to a local folder to see whether the Overlaybar reads it OK there?  You should be able to do this with keyboard only to avoid hovering it.

**Comment by yurutken** [comment]
1. I downloaded all files to local, even with non standart character folder names, there was no problem.It showed file size -- dimensions correctly.
2. I put same files to another FTP, there was no problem again.
3. Hanging sometimes happens just after showing dimensions of file, everytime i try to open same folder it hangs but sometimes shows file size and dimension but hang also.

**Comment by jeremypw** [comment]
@yurutken Thanks for the additional information. 
>I put same files to another FTP, there was no problem again.
What do you mean by "another FTP"?  A different FTP server  but still using Files?

I'll have a look at the Overlaybar code to see if I can see somewhere it might block.  We may have to inhibit the overlay for FTP servers or have an option to do so.

**Comment by yurutken** [comment]
Yes a different FTP server, i copied same JPG files to it, i even open an FTP server with an Android telephone and copied files to it. It didnt make any problem.
I think that particular FTP server is somehow causing problem, but whatever the problem with FTP server is "Files" must not stop responding in all situations. If i can i will also try to look Overlaybar code. Thanks for your time also.

**Comment by yurutken** [comment]
in "LIST VIEW" mode there is no problem no hang.
"ICON" and "MILLER_COLUMNS" modes cause hang.

**Comment by jeremypw** [comment]
The overlaybar did not used to show in ListView - do you still have that version? Or is the Overlay showing in all views?

**Comment by yurutken** [comment]
Yes, actually Overlay bar is not used in ListView. But i thought i am using last version ?

dpkg -s pantheon-files

Package: pantheon-files
Status: install ok installed
Priority: optional
Section: x11
Installed-Size: 3055
Maintainer: Cody Garver <cody@elementary.io>
Architecture: amd64
Version: 4.4.2+r3786+pkg92~ubuntu5.1.2.1
Depends: gvfs, libpantheon-files-core0 (= 4.4.2+r3786+pkg92~ubuntu5.1.2.1), libpantheon-files-widgets0 (= 4.4.2+r3786+pkg92~ubuntu5.1.2.1), tumbler, dconf-gsettings-backend | gsettings-backend, libc6 (>= 2.4), libcairo2 (>= 1.2.4), libcloudproviders0 (>= 0.3.0), libdbusmenu-glib4 (>= 0.4.2), libgdk-pixbuf2.0-0 (>= 2.23.0), libgee-0.8-2 (>= 0.8.3), libgit2-glib-1.0-0 (>= 0.0.6), libglib2.0-0 (>= 2.39.4), libgranite5 (>= 5.3.1+r1684+pkg113~ubuntu5.1.2.1), libgtk-3-0 (>= 3.21.5), libpango-1.0-0 (>= 1.20.0), libplank1 (>= 0.11.89), libsqlite3-0 (>= 3.5.9), libunity9 (>= 3.4.6), libzeitgeist-2.0-0 (>= 0.9.9)
Recommends: contractor, exfat-fuse, gvfs-backends, tumbler-plugins-extra
Description: File manager for the Pantheon desktop
 Pantheon Files is a modern file manager originally
 designed for the Pantheon desktop environment.
 .
 It features:
  - Column view to effortlessly traverse complex folder hierarchies,
    in addition to Icon and Tree views
  - Contextual actions on files and folders provided by Contractor
  - Transparent support for remote servers and network shares
  - Compliance with elementary HIG, e.g. lets you restore closed tabs
  - Session persistence - take up where you left!
Homepage: https://github.com/elementary/files

**Comment by jeremypw** [comment]
Ah, yes - that commit has not been released yet - sorry.

**Comment by yurutken** [comment]
Same ftp in another new folder it also hangs, only when hovered on JPG files.
It seems that for some time i will have to use another files manager.

**Comment by jeremypw** [comment]
Closing as it seems related to the hover-info overlay which is no longer.  Please feel free to re-open if recurs on the latest version.
